### ISSUE TITLE
Namespace heads up

### ISSUE THREAD

**Issue by hadley** [jrnold/ggthemes#35]
In the next version of ggplot2, I import and re-export `scales::alpha()`, `grid::unit()` and `grid::arrow()`. Because you're importing all of those namespaces, that gives:

```R
library(ggthemes)
#> Warning: replacing previous import by 'grid::arrow' when loading 'ggthemes'
#> Warning: replacing previous import by 'grid::unit' when loading 'ggthemes'
#> Warning: replacing previous import by 'scales::alpha' when loading
#> 'ggthemes'
```

**Comment by jrnold** [comment]
Thanks. I've removed some of the extraneous importing, and I'll remove the import of `grid::unit` when the new version of ggplot2 comes out.

**Comment by jrnold** [comment]
Fixed in #37 and earlier commits.
